src/components/Feed/Feed.tsx

```diff
@@ -0,0 +1,67 @@
+import React, {useEffect, useState} from 'react';
+import ArticleCard from "./ArticleCard";
+import {iArticle} from "../../commonUse/types/iArticle";
+import fetchData from "../../commonUse/functions/fetchData";
```

Comment:
Could be configured to use absolute imports like `"commonUse/types.."`
Would look better